src/pages/about.astro

```diff
@@ -6,14 +6,23 @@ import JobExperience from "../components/AstroCompoent/JobAndExperience/JobExper
 import authorConfig from "../config/info";
 export const prerender = true;
 import Footer from "../components/AstroCompoent/footer/Footer.astro";
+import { featureFlags } from "../config/featureFlag";
+
+try {
+  if (!featureFlags.showAbout) {
+    return Astro.redirect("/403");
+  }
+} catch (error) {
+  return Astro.redirect("/400");
+}
 ---
 
 <!doctype html>
 <html lang="en">
   <head>
     <BaseHead
-      title={authorConfig.SiteName}
-      description={authorConfig.SiteDescription}
+      title={authorConfig?.SiteName ?? ""}
+      description={authorConfig?.SiteDescription ?? ""}
```

Comment:
Yeah, I don't want this all the data will be coming from the info.ts and if there is non then it will be blank to inform the user that please provide the data.